backend/coins/eth/account.go

```diff
@@ -334,6 +349,35 @@ func (account *Account) outgoingTransactions(allTxs []*accounts.TransactionData)
 	return transactions, nil
 }
 
+func (account *Account) updateNextNonce() error {
+	// Nonce to be used for the next tx, fetched from the ETH node. It might be out of date due to
+	// latency, which is addressed below by using the locally stored nonce.
+	nodeNonce, err := account.coin.client.PendingNonceAt(context.TODO(), account.address.Address)
+	if err != nil {
+		return err
+	}
+	account.nextNonce = nodeNonce
+
+	confirmedTransactions, err := account.confirmedTransactions()
+	if err != nil {
+		return errp.WithStack(err)
+	}
+	outgoingTransactions, err := account.outgoingTransactions(confirmedTransactions)
+	if err != nil {
+		return errp.WithStack(err)
+	}
```

Comment:
This is new behavior - the transactions are now fetched from etherscan while filling out the send form. I believe the `newTx()` function is called every time the user updates the send form :see_no_evil: So it's one more call everytime the user updates the send form (which is when `newTx()` is called). With the rate limit of ~4calls/sec it could also cause a noticeable delay when filling out the form (I didn't test).

Instead, you could use `account.transactions`.